addons/cartel/config.cpp

```diff
@@ -28,8 +28,8 @@ class CfgPatches {
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"tacu_main"};
-        author = ECSTRING(Cartel,Author);
-        authors[] = {ECSTRING(Cartel,Author)};
+        author = "Tyrone";
```

Comment:
This one should be `ECSTRING(main,Author)`.

Reply:
Ah, my bad. Both "author" and "authors"? (Will correct the other ones as well then)

Reply:
No, just `author`. That way you have author of component in-game show up as "TAC Units Team", but the list of all authors show names of individuals.

Reply:
Gotcha, will update the other ones as well to have ECSTRING.
@TyroneMF sorry about that, edit this one back as JJ mentioned, then everything should be good to be merged I believe.

Reply:
It's just consistency, it's how we do it in CBA, ACE3, Theseus Services...